api/tests/c/indigo-test-data-issue-269.cpp

```diff
@@ -0,0 +1,1061 @@
+extern "C" const char* issue269test_RGroupsWarning = R"(
```

Comment:
There are warnings in g++ for this file. It is worth to move the test into the python tests